src/migrations/20220411103724-add-legal-value-description.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        -- 1. Convert '1,2,3'::text to "1,2,3"::json
+        alter table context_fields
```

Comment:
Would prefer if we follow the convention in previous migrations of uppercasing the ALTER TABLE / SELECT statements here. 

As for the migration itself, I _think_ it looks good, but I'd like @chriswk and @sighphyre  to weigh in here.

Reply:
The SQL seems sane, good catch on the casing to follow convention